src/Build/Logging/BinaryLogger/BinaryLogger.cs

```diff
@@ -139,9 +142,23 @@ public void Initialize(IEventSource eventSource)
 
             binaryWriter.Write(FileFormatVersion);
 
+            LogInitialInfo();
+
             eventSource.AnyEventRaised += EventSource_AnyEventRaised;
         }
 
+        private void LogInitialInfo()
```

Comment:
Inline this?

Reply:
I prefer the compile to do that, and keep the functions to improve readability.